addons/point_of_sale/static/src/js/models.js

```diff
@@ -1875,6 +1885,9 @@ export class Orderline extends PosModel {
     get_price_extra() {
         return this.price_extra;
     }
+    updateSavedQuantity() {
+        this.quantity_saved = this.quantity;
+    }
```

Comment:
This shouldn't be needed because the saved quantity will always be up to date whenever you load an order